addons/account/tests/common.py

```diff
@@ -529,7 +529,7 @@ def setUpClass(cls, chart_template_ref=None):
             'categ_id': cls.env.ref('product.product_category_all').id,
         })
 
-        cls.bank_journal_euro = cls.env['account.journal'].create({'name': 'Bank', 'type': 'bank', 'code': 'BNK67'})
+        cls.bank_journal_euro = cls.env['account.journal'].create({'name': 'Bank', 'type': 'bank', 'code': 'BNK67', 'alias_name': 'Bank0'})
```

Comment:
We only need an alias for `sale` and `purchase` journals

Reply:
@william-andre if I don't do these changes, the runbot tests will fail: https://runbot.odoo.com/runbot/build/9697860

Reply:
That's because you must change the way the business code changes in order not to change this test...